src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java

```diff
@@ -80,7 +80,7 @@
    * Test switch to log all bytes that come across the wire.
    * Helpful when parsing fails before certain bytes hit the trace logs.
    */
-  public static boolean LogAllWireBytes = false;
+  public static boolean logAllWireBytes = false;
```

Comment:
I also thought about it, but this field is used outside this class. I can add getter and setter for this field or keep it as now. What do you prefer?